docker-compose.yml

```diff
@@ -0,0 +1,102 @@
+# Docker compose for API ML modulith
+version: '3.8'
+
+services:
+  apiml:
+    image: ghcr.io/zowe/apiml:latest
+    ports:
+      - "10010:10010" # External access
+      - "10011:10011" # External access
+      - "5130:5130" # For Java remote debugging
+    volumes:
+      - /Users/pc891986/Projects/api-layer-v3/config/docker/api-defs:/api-defs
```

Comment:
Is it pushed intentionally? See the userId.

Reply:
it's an example only, removed